applications/RomApplication/tests/structural_static_test_files/PGROM/ProjectParametersPGROM.json

```diff
@@ -0,0 +1,108 @@
+                    "value": [0.0,0.0,0.0],
+                    "interval": [0.0,"End"]
+                }
+            }
+        ],
+        "loads_process_list": [{
+            "python_module" : "assign_vector_by_direction_to_condition_process",
+            "kratos_module" : "KratosMultiphysics",
+            "check"         : "DirectorVectorNonZero direction",
+            "Parameters"    : {
+                "model_part_name" : "Structure.PointLoad2D_Load_on_points_Auto1",
+                "variable_name"   : "POINT_LOAD",
+                "modulus"         : 50.0,
+                "direction"       : [1.0,0.0,0.0],
+                "interval"        : [0.0,1.0]
+            }
+        },{
+            "python_module" : "assign_vector_by_direction_to_condition_process",
+            "kratos_module" : "KratosMultiphysics",
+            "check"         : "DirectorVectorNonZero direction",
+            "Parameters"    : {
+                "model_part_name" : "Structure.LineLoad2D_InterfaceStructure",
+                "variable_name"   : "LINE_LOAD",
+                "modulus"         : 50.0,
+                "direction"       : [1,0.0,0.0],
+                "interval"        : [1.0,2.0]
+            }
+        }],
+        "list_other_processes": []
+    },
+    "output_processes": {
+        "gid_output" : [{
+            "python_module" : "gid_output_process",
+            "kratos_module" : "KratosMultiphysics",
+            "process_name"  : "GiDOutputProcess",
+            "help"          : "This process writes postprocessing files for GiD",
+            "Parameters"    : {
+                "model_part_name"        : "Structure",
+                "output_name"            : "Simple_beam",
+                "postprocess_parameters" : {
+                    "result_file_configuration" : {
+                        "gidpost_flags"               : {
+                            "GiDPostMode"           : "GiD_PostBinary",
+                            "WriteDeformedMeshFlag" : "WriteDeformed",
+                            "WriteConditionsFlag"   : "WriteConditions",
+                            "MultiFileFlag"         : "SingleFile"
+                        },
+                        "file_label"                  : "step",
+                        "output_control_type"         : "step",
+                        "output_interval"             : 1,
+                        "body_output"                 : true,
+                        "node_output"                 : false,
+                        "skin_output"                 : false,
+                        "plane_output"                : [],
+                        "nodal_results"               : ["DISPLACEMENT"],
+                        "gauss_point_results"         : [],
+                        "nodal_nonhistorical_results" : []
+                    },
+                    "point_data_configuration"  : []
+                }
+            }
+        }],
```

Comment:
By doing this you're always printing the output. This should be optional, defaulted to false and only active for debugging.

Same comment applies in the other tests.